### ISSUE TITLE
Unhandled Runtime Error after Login. 

### ISSUE THREAD

**Issue by Pintiboy** [pashabitz/informal#1]
Getting the following error:
![image](https://github.com/user-attachments/assets/fb70fc77-8b31-41e0-80ef-676f51e340e8)

**Comment by pashabitz** [comment]
Does this still happen? Unable to reproduce at the moment.

**Comment by Pintiboy** [comment]
Yes, tried it again, this time on a different machine. Happens also here.
